src/DurableFunctions.TypedInterfaces/README.md

```diff
@@ -207,7 +207,7 @@ For the moment, generated code will not show up intellisense unless the namespac
 
 To use the generated interfaces, and have intellisense available for the generated types, you must manually include the using statement:
 ```csharp
-using using DurableFunctions.TypedInterfaces;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.TypedInterfaces;
```

Comment:
I noticed there is a reference to another namespace shortly above this.
> For the moment, generated code will not show up intellisense unless the namespace containing the code is added to the file you are trying to use them in. Generated code is placed in the namespace `Microsoft.Azure.Webjobs.Extensions.DurableTask.Generated`.

Is this correct?